src/com/projectkorra/projectkorra/firebending/FireShield.java

```diff
@@ -240,20 +250,12 @@ public void setShieldDuration(final long duration) {
 		this.shieldDuration = duration;
 	}
 
-	public long getInterval() {
-		return this.interval;
-	}
-
-	public void setInterval(final long interval) {
-		this.interval = interval;
-	}
-
 	public double getRadius() {
-		return this.radius;
+		return this.shieldRadius;
 	}
 
 	public void setRadius(final double radius) {

```

Comment:
This method and its parameter should be renamed to better reflect the specific variable it modifies. Something like `setShieldRadius` would be more fitting.